src/main/groovy/org/mockito/release/internal/gradle/util/PomCustomizer.groovy

```diff
@@ -82,13 +85,14 @@ class PomCustomizer {
                 d.appendNode('url', "https://github.com/${split[0]}")
             }
 
-            def contributors = root.appendNode('contributors')
-            rootProject.ext.pom_contributors.each {
-                def split = it.split(':')
-                assert split.length == 2
-                def c = contributors.appendNode('contributor')
-                c.appendNode('name', split[1])
-                c.appendNode('url', "https://github.com/${split[0]}")
+            def contributorsNode = root.appendNode('contributors')
+            def fileName = Contributors.getAllProjectContributorsFileName(project.getBuildDir())
+            AllProjectContributorsReader reader = AllProjectsContributorsProvider.allProjectContributorsReader()
```

Comment:
Good point, we need to make the implementation more flexible. It should be easy to just configure the users explicitly if the users want such behavior. I made a comment about it in the code.

Thanks for review!